.github/workflows/build_and_test.yml

```diff
@@ -154,15 +155,10 @@ jobs:
           name: gradle-test-report-${{ matrix.os }}
           path: '**/build/reports/'
           retention-days: 1
-      - name: Code coverage report
```

Comment:
But now code coverage is not run even in the main repository? When you have removed a step

Reply:
Code coverage is already executed in `build_and_test_with_code_coverage` job:

https://github.com/saveourtool/diktat/blob/54a803081c98db78adbb21bb20ab185805314000/.github/workflows/build_and_test.yml#L68-L72

And it was duplicated in the `build_and_test` job:

https://github.com/saveourtool/diktat/blob/54a803081c98db78adbb21bb20ab185805314000/.github/workflows/build_and_test.yml#L157-L161

This is why I removed it from here. It was also running/uploading for every OS in the job's matrix. But maybe that was intentional.

Reply:
Got it, then good, thanks! 
Actually not intentional 😄